Client/Frontend/Browser/MailProviders.swift

```diff
@@ -8,41 +8,74 @@ import Shared
 // mailto headers: subject, body, cc, bcc
 
 protocol MailProvider {
-    var beginningScheme: String {get set}
+    var urlFormat: String {get set}
     var supportedHeaders: [String] {get set}
     func newEmailURLFromMetadata(_ metadata: MailToMetadata) -> URL?
 }
 
-private func constructEmailURLString(_ beginningURLString: String, metadata: MailToMetadata, supportedHeaders: [String], bodyHName: String = "body", toHName: String = "to") -> String {
-    var lowercasedHeaders = [String: String]()
-    metadata.headers.forEach { (hname, hvalue) in
-        lowercasedHeaders[hname.lowercased()] = hvalue
-    }
+extension MailProvider {
+    fileprivate func constructEmailURLString(_ urlFormat: String,
```

Comment:
nit: any particular reason for fileprivate vs private 🤔

Reply:
If we make this function private the classes implementing the protocol have no access to it and the builds are failing.